packages/harmony/src/storybook/components/Lockup.tsx

```diff
@@ -0,0 +1,26 @@
+import { Flex, Text, Link } from 'components'
+import type { IconComponent } from 'components/typography/Icons/Icon'
+
+type LockupProps = {
+  icon: IconComponent
+  subtitle: string
+  description: string
+  link?: string
+}
+
+export const Lockup = (props: LockupProps) => {
```

Comment:
What's a lockup?

Reply:
Not sure... it's the name that is in Figma, let's ask Sammie

Reply:
yea im not sure what lockup is either lol. Maybe block or something?

Reply:
we got it renamed to "CardLink"